packages/web-components/fast-foundation/src/scroller/scroller.template.ts

```diff
@@ -0,0 +1,37 @@
+import { html, ref } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../patterns";
+import { Scroller } from "./scroller";
+
+/**
+ * @public
+ */
+export const ControlsTemplate = html<Scroller>`
+    <div ${ref("previousFlipper")} class="scroll scroll-prev">
+        <div @click="${x => x.scrollToPrevious()}" class="scroll-cntrl">
+            <slot name="previous-flipper"></slot>
```

Comment:
I had considered having a default. It would be nice if it just came with a flipper rather than having to add what would likely be the most commonly used. But they may not want flippers at all. It may be controlled by an outside handler or script. Plus I wasn't sure if this was going to be a problem moving it from FAST to FluentUI and to it's final destination, Edge feeds.

Reply:
Ah yes, if there is potential that the implementation doesn't want flippers at all that could be an issue.
How often would this be the case?
If you add a part name the the previousFlipper div then they could just hide that if they don't want flippers.

I know for FluentUI there was talk of adding methods to allow for a different default to be added to the template, but not sure where that conversation left off.